runio.js

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env node
+const fs = require('fs');
+const path = require('path');
+const { stopOnFail, chdir, git, copy, exec, replaceInFile, npmRun, npx, writeToFile, runio } = require('runio');
+
+stopOnFail();
+
+module.exports = {
+  async docs() {
+    await Promise.all([
+      this.docsHelpers(),
+      this.docsPlugins(),
+      this.docsExternalHelpers(),
+    ]);
+
+  },
+
+  async docsPlugins() {
+    // generate documentation for plugins
+    await npx('documentation build lib/plugin/*.js -o docs/plugins.md -f md --shallow --markdown-toc=false --sort-order=alpha');
+    await replaceInFile('docs/plugins.md', cfg => {
+      cfg.replace(/^/,'---\nid: plugins\ntitle: Plugins\n---\n\n')
+    })
+  },
+
+  async docsExternalHelpers() {
+    // generate documentation for helpers outside of main repo
+    console.log('Building @codecepjs/detox helper docs');
+    let helper = 'Detox';
+    await npx(`documentation build node_modules/@codeceptjs/detox-helper/${helper}.js -o docs/helpers/${helper}.md -f md --shallow --markdown-toc=false --sort-order=alpha `);
+    await writeToFile(`docs/helpers/${helper}.md`, cfg => {
+      cfg.line(`---\nid: ${helper}\ntitle: ${helper}\n---\n\n`);
+      cfg.textFromFile(`docs/helpers/${helper}.md`);
+    });
+  },
+
+  async docsHelpers() {
+    // generate documentation for helpers
+    const files = fs.readdirSync('lib/helper').filter(f => path.extname(f) === '.js');
+
+    const partials = fs.readdirSync('docs/webapi').filter(f => path.extname(f) === '.mustache');
+    const placeholders = partials.map(file => `{{> ${path.basename(file, '.mustache')} }}`);
+    const templates = partials
+    .map(file => fs.readFileSync('docs/webapi/' + file).toString())
+    .map(template => template.replace(/^/m, '\n   * '));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 6 spaces but found 4. (indent)](https://app.codacy.com/app/DavertMik/CodeceptJS/pullRequest?prid=4496293)